tests/pipelines/stable_diffusion_xl/test_stable_diffusion_xl.py

```diff
@@ -261,6 +261,43 @@ def test_stable_diffusion_xl_offloads(self):
         assert np.abs(image_slices[0] - image_slices[1]).max() < 1e-3
         assert np.abs(image_slices[0] - image_slices[2]).max() < 1e-3
 
+    def test_stable_diffusion_xl_img2img_prompt_embeds_only(self):
```

Comment:
For completeness, do we want to test the prompt embed case where prompt is passed in as a string? Currently all tests related to prompt embeds pass in the prompts as a list.

Reply:
Hmm we always normalize single strings to a list in the code, so maybe not necessary here?